app/queries/pin_filter_query.rb

```diff
@@ -9,7 +9,17 @@ def initialize(keywords)
   end
 
   def filtered
-    if surgeons.present?
+    if surgeons.present? && procedures.present?
```

Comment:
this has become pretty gnarly but i was punting it onto https://github.com/mooreniemi/Transbucket_Rails/issues/45 for the moment :/